src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyName.cs

```diff
@@ -164,6 +178,11 @@ public void SetPublicKey(byte[]? publicKey)
                 _flags |= AssemblyNameFlags.PublicKey;
         }
 
+        private byte[]? ComputePublicKeyToken()
+        {
+            return AssemblyNameHelpers.ComputePublicKeyToken(_publicKey);
```

Comment:
I think you can just call `AssemblyNameHelpers.ComputePublicKeyToken` directly now. No need to wrap it.